### ISSUE TITLE
Make the cost of gas assets  visible (and possibly adaptable)

### ISSUE THREAD

**Issue by MarjoleinSchwachofer** [quintel/etmoses#1344]
Why are in the gas asset list the technical_lifetime ,  initial_investment  and  om_costs_per_year  not visible? This should give more insight in the costs, and give an idea where the costs in the businesscase come from. The technical lifetime with the building year give also more insights in the equity and cumalative investment curve.

Explanation about these curves is also missing in the documentation. 

Currently, the user also cannot change these attributes in the front-end. Why is it that for the gas assets these attributes above cannot be changed, and for all the other elements (e.g. electricity and heat) it is possible? I think this should also be the case for gas, but with the standard values filled in.

**Comment by ChaelKruip** [comment]
> Why are in the gas asset list the technical_lifetime , initial_investment and om_costs_per_year not visible?  

This has been discussed extensively with @Sanne512. The reasons are:
*  Almost all users will not know more accurate number than those provided by the SO (Alliander in this case)
* The gas asset table is already quite wide, adding more columns will make it necessary to scroll horizontally

> This should give more insight in the costs, and give an idea where the costs in the businesscase come from.

I have added overview [tables to the wiki](https://github.com/quintel/etmoses/wiki/Gas-network) to improve the transparency.

**Comment by ChaelKruip** [comment]
> I think this should also be the case for gas, but with the standard values filled in.

There is no reason (except those mentioned above) to not allow the user to change these numbers. @MarjoleinSchwachofer what do you suggest interface-wise? Where should the user be given the option to edit the costs?

**Comment by Sanne512** [comment]
> The gas asset table is already quite wide, adding more columns will make it necessary to scroll horizontally

I agree with this, but didn't we also discuss that it would be nice to give users the option to change the costs if they wanted to in a similar lay out to the technology table? 

It would be nice if users could change the costs, because they have to insert the costs for the electricity infrastructure any way. If users have access to these numbers they also possibly have access to costs for the gas infrastructure. I also think that it would make the model stronger, because in my opinion the strength lays in the fact that the user can adjust the model in detail if they have access to these numbers.

**Comment by MarjoleinSchwachofer** [comment]
> Almost all users will not know more accurate number than those provided by the SO (Alliander in this case)

This is also the case for electricity and heat. I agree with Sanne that a strength of the model is that you can adjust  parameters if you think you have better values. And that you can see them directly, so that you know where the item in the business case comes from.

>The gas asset table is already quite wide, adding more columns will make it necessary to scroll horizontally
 

> @MarjoleinSchwachofer what do you suggest interface-wise? Where should the user be given the option to edit the costs?

I liked the way it is presented in the heat asset list.  And on my screen there is enough room available to the left and right of the screen, but I do not know if there are restrictictions?

**Comment by ChaelKruip** [comment]
> I liked the way it is presented in the heat asset list. And on my screen there is enough room available to the left and right of the screen, but I do not know if there are restrictictions?

The gas asset table is already 1200 pixels wide:
![etmoses](https://cloud.githubusercontent.com/assets/1303760/17396733/011fdb78-5a35-11e6-962b-44d1369e580d.png)

This is already quite wide if you ask the internet. See [http://www.hobo-web.co.uk/best-screen-size/](http://www.hobo-web.co.uk/best-screen-size/) or [http://ux.stackexchange.com/questions/42828/how-wide-should-a-responsive-designed-website-be](http://ux.stackexchange.com/questions/42828/how-wide-should-a-responsive-designed-website-be) to pick a few. 

Having said that, there is no hard limit. Adding three more columns is not impossible but it will not make the table easier to read. Perhaps we could hide some columns by default and allow the user to expand the table with a button?

@antw @grdw @dennisschoenmakers what do you think would be a 'nice' way to give the user the option to change costs for gas asset?

**Comment by grdw** [comment]
> what do you think would be a 'nice' way to give the user the option to change costs for gas asset?

My best guess is to switch from a table view to a 'list' view with multiple rows of inputs per list-item. A bit like the technology matrix in a way

````
Pressure level:  0.125                                     + (Add new gas asset)
------------------------------------------------------------
Asset:              [    v ]  Type:               [    v ] X (Remove a gas asset)
Quantity:           [      ]  Building year:      [      ]
Stakeholder:        [      ] Technical lifetime:  [      ]
Initial investment: [      ]  OM costs per year:  [      ]
------------------------------------------------------------
Asset:              [    v ]  Type:               [    v ] X
Quantity:           [      ]  Building year:      [      ]
Stakeholder:        [      ] Technical lifetime:  [      ]
Initial investment: [      ]  OM costs per year:  [      ]
------------------------------------------------------------
Asset:              [    v ]  Type:               [    v ] X
Quantity:           [      ]  Building year:      [      ]
Stakeholder:        [      ] Technical lifetime:  [      ]
Initial investment: [      ]  OM costs per year:  [      ]

Pressure level:  4                                         +
------------------------------------------------------------
Asset:              [    v ]  Type:               [    v ] X
Quantity:           [      ]  Building year:      [      ]
Stakeholder:        [      ] Technical lifetime:  [      ]
Initial investment: [      ]  OM costs per year:  [      ]
------------------------------------------------------------
Asset:              [    v ]  Type:               [    v ] X
Quantity:           [      ]  Building year:      [      ]
Stakeholder:        [      ] Technical lifetime:  [      ]
Initial investment: [      ]  OM costs per year:  [      ]

````
